Formula/c/cabin.rb

```diff
@@ -0,0 +1,58 @@
+class Cabin < Formula
+  desc "Package manager and build system for C++"
+  homepage "https://github.com/cabinpkg/cabin"
+  license "Apache-2.0"
+  head "https://github.com/cabinpkg/cabin.git", branch: "main"
+
+  stable do
+    url "https://github.com/cabinpkg/cabin/archive/refs/tags/0.10.1.tar.gz"
+    sha256 "ba6a4b2df33ab9bb4eb5923e550bfe81725658fa6a647be5b0b4508ebfe7acc8"
```

Comment:
0.11.0 is out - https://github.com/cabinpkg/cabin/releases/tag/0.11.0

This includes all the renaming https://github.com/cabinpkg/cabin/commit/84dac2eea82db8ac4572db126aa699b37745dab6